docs/examples/training.ts

```diff
@@ -52,62 +67,26 @@ async function loadSimpleFaceData (task: Task): Promise<data.DataSplit> {
   const youngFolders = ['child']
   const oldFolders = ['adult']
 
-  const youngFiles = youngFolders.flatMap(folder => {
-    return filesFromFolder(dir, folder)
-  })
-
-  const oldFiles = oldFolders.flatMap(folder => {
-    return filesFromFolder(dir, folder)
-  })
+  const youngFiles = (await Promise.all(youngFolders.map(async folder => await filesFromFolder(dir, folder)))).flat()
+  const oldFiles = (await Promise.all(oldFolders.map(async folder => await filesFromFolder(dir, folder)))).flat()
 
   const filesPerFolder = [youngFiles, oldFiles]
 
   const labels = filesPerFolder.flatMap((files, index) => Array(files.length).fill(index))
   const files = filesPerFolder.flat()
 
-  return await new node.data.NodeImageLoader(task).loadAll(files, { labels: labels })
+  return await new NodeImageLoader(task).loadAll(files, { labels })
 }
 
 async function loadTitanicData (task: Task): Promise<data.DataSplit> {
   const files = ['../../example_training_data/titanic_train.csv']
   const titanicTask = defaultTasks.titanic.getTask()
-  return await new node.data.NodeTabularLoader(task, ',').loadAll(files, {
+  return await new NodeTabularLoader(task, ',').loadAll(files, {
     features: titanicTask.trainingInformation.inputColumns,
     labels: titanicTask.trainingInformation.outputColumns,
     shuffle: false
   })
 }
 
-async function startServer (): Promise<[http.Server, URL]> {
-  const uninit_server = new DiscoServer()
-  await uninit_server.addDefaultTasks()
-
-  const server: http.Server = uninit_server.serve(8000)
-  await new Promise((resolve, reject) => {
-    server.once('listening', resolve)
-    server.once('error', reject)
-    server.on('error', console.error)
-  })
-
-  let addr: string
-  const rawAddr = server.address()
-  if (rawAddr === null) {
-    throw new Error('unable to get server address')
-  } else if (typeof rawAddr === 'string') {
-    addr = rawAddr
-  } else if (typeof rawAddr === 'object') {
-    if (rawAddr.family === '4') {
-      addr = `${rawAddr.address}:${rawAddr.port}`
-    } else {
-      addr = `[${rawAddr.address}]:${rawAddr.port}`
-    }
-  } else {
-    throw new Error('unable to get address to server')
-  }
-
-  return [server, new URL('', `http://${addr}`)]
-}
-
-
 // You can run this example with "npm start" from this folder
-main().catch(console.error)
+main('titanic').catch(console.error)
```

Comment:
I feel this parameter can be easily missed, I think a constant in capslock at the top of the file would be more visible but less clean for sure

Reply:
clearly, I lost myself in wanting to scope everything :sweat_smile:
adding a screaming constant at the begin _of the function_ would be perfect